src/DIRAC/AccountingSystem/DB/AccountingDB.py

```diff
@@ -1149,7 +1151,16 @@ def __queryType(
     if sqlLinkList:
       cmd += " AND %s" % " AND ".join(sqlLinkList)
     if groupFields:
-      cmd += " GROUP BY %s" % (groupFields[0] % tuple(groupFields[1]))
+      if len(groupFields[1]) == 1:
+        testGroupFields = " %s" % selectFields[0] % tuple(realFieldList)
+        testGroupFieldsList = testGroupFields.split(",")
+        realGroupFields = ()
+        for testGroupFields in testGroupFieldsList:
+          if "sum" not in testGroupFields.lower():
+            realGroupFields += (testGroupFields.strip(),)
+        cmd += " GROUP BY " + ','.join(realGroupFields)
```

Comment:
IIUC `realGroupFields` can be empty
```suggestion
        if realGroupFields:
          cmd += " GROUP BY " + ','.join(realGroupFields)
```